specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json

```diff
@@ -862,15 +862,27 @@
         "purchaseProperties": {
           "$ref": "#/definitions/PurchaseRequest"
         },
-        "lockedPriceTotal": {
+        "pricingCurrencyTotal": {
```

Comment:
This is categorized as a breaking change and would require a new api version
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo

Reply:
Old property ‘lockedPriceTotal’ and new properties ‘billingCurrencyTotal’ and ‘pricingCurrencyTotal’ are part of the same work which is not released to Prod yet.
Nobody has taken a dependency on these properties as of yet. The change in properties has been made after an internal review.
So, this is not a breaking change.

Reply:
This is a documentation issue as the properties being changed haven't gone into the runtime of the service. This is also part of a read-only response property so it's non breaking anyways.